### DISCUSSION TITLE
Import to update from file

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#5832] [Archive]
I see the new endpoint to import from a CSV file, can you however import to replace data based on the primary key?

so for example my table has
id.    name.     specs
1.      bob.        acme  
2.     dan.         bbc
3.     Ean.         cnn

and I import a CSV file with 

id.    specs
2.     cat
3.     dog

my final result in the collection 

id.    name.     specs
1.      bob.        acme  
2.     dan.         cat
3.     Ean.         dog

**Comment by benhaynes** [comment]
Yes, it uses an upsert. If you include the PK it will update existing records. Just remember to follow the exact same format as our export endpoint data.
